architecture/consumption-architecture-dependency-management.md

```diff
@@ -0,0 +1,148 @@
+![overview](./images/OpenSSF_Practitioner_Framework%20_Overview.jpg)
+
+OpenSSF and its peer foundations provide OSS technologies to raise security awareness through education and training, establish best software development practices, guides, security standards, frameworks, specifications, and provide security tools to improve open source software dependency management. The security tools are built on top of the ecosystem support to make open source software ecosystems more secure.
+
+Every layer in the diagram provides support to the layers above it, relies on the underneath layers for foundational support. 
+
+### Dependency Ingestion Policy and Enforcement
+
+![State](./images/OpenSSF_Practitioner_Framework_state.jpg)
+![policy and enforcement](./images/OpenSSF_Practitioner_Framework%20_Ingestion_Policy_and_Enforcement.jpg)
+
+To sustain OSS dependency management at scale, with accuracy and efficiency, we need technical solutions that can define software ingestion policies and enforce those policies. For example, if an open source software has not been maintained actively, the software should not be introduced into your code base. 
+
+The diagram has illustrated that dependency ingestion policy and enforcement is a very green field on OSS producer side (This may change soon with Minder joining OpenSSF as an OSS project). The technologies in this layer will evolve as we gain more [Security Insights into OSS Projects](#security-insights-into-oss-projects). On the customer side, organizations have their own governance and technologies in managing OSS dependencies. 
+* [giituf](https://github.com/gittuf/gittuf) is an OpenSSF [sandbox](https://github.com/ossf/tac/blob/main/process/project-lifecycle.md#stages---definitions--expectations) project hosted in the [Supply Chain Integrity Working Group](https://github.com/ossf/wg-supply-chain-integrity). It provides a security layer over Git. It's in active development phase and conducts policy validation. 
+* [Witness](https://github.com/in-toto/witness) is a CNCF project under the [in-toto](https://github.com/in-toto) umbrella that protects software supply chain integrity. Witness comes with an OPA policy engine that verifies the software attestation and ensures that your dependency was handled safely from source to deployment. Witness in-toto attestation feature is available for CI systems on OSS producer side and consumer side  
+* [Repository Service for TUF(RSTUF](https://github.com/repository-service-tuf/repository-service-tuf) is an OpenSSF [incubating](https://github.com/ossf/tac/blob/main/process/project-lifecycle.md#stages---definitions--expectations) project in the [Securing Software Repositories Working Group](https://github.com/ossf/wg-securing-software-repos). The verification component of RSTUF provides metadata verification to ensure the software that's being ingested is as expected. RSTUF metadata verification can be used by both CI systems on OSS producer side and consumer side.    
```

Comment:
I definitely think that RSTUF plays a role here but, I'm struggling to provide a different description for this context. I'm sure that @kairoaraujo can help. ☺️

Reply:
@kairoaraujo, the document has the latest update based on our recommendation.